tests/macos/xcodeproj/NonArcObject.m

```diff
@@ -1,4 +1,11 @@
 #import "NonArcObject.h"
 
 @implementation NonArcObject
+#if REQUIRED_DEFINED_FLAG
 @end
+#endif
+
+#if FLAG_WITH_VALUE_ZERO
```

Comment:
I'm not sure why this code is compiling .... it seems like `FLAG_WITH_VALUE_ZERO` has been mistakenly specified due to bug I noted above, where everything is getting appended to `defines_without_equals_sign`

Reply:
we have discussed and resolved this